lib/appinsights/middlewares/track_request.rb

```diff
@@ -0,0 +1,20 @@
+require 'application_insights'
+
+module AppInsights
+  class TrackRequest
+    def initialize(app, instrumentation_key, ignore_prefixes = [], buffer_size = 500, send_interval = 60)
+      @app = app
+      @ignore_prefixes = ignore_prefixes
```

Comment:
nitpick: from the name I though this was a boolean, `ignore_prefixes = true/false`
 `prefixes_to_ignore`,  `ignore_prefix_list`, ...